version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "8.0.0"
+version in ThisBuild := "8.1.0"
```

Comment:
Technically maybe a minor version. But given that we do a major bump in Kafka deps, maybe worth a major bump as well?